packages/redux/src/bags/selectors.ts

```diff
@@ -619,3 +626,88 @@ export const isProductInBag = (
 
   return bagItems.some(bagItem => bagItem.product?.id === Number(productId));
 };
+
+/**
+ * Retrieves a specific bag operation by its id,.
+ *
+ * @param state - Application state.
+ * @param bagOperationId - Unique identifier of the operation.
+ *
+ * @returns - Bag operation entity for the given id.
```

Comment:
Done!

Reply:
Nitpicking here, but the description contains a `..` instead of a `.`. Most likely it was because of the terrible mac keyboard that registered 2 key presses all the time. Can you remove the lingering dot from there (or if you prefer I can do it real quick, just want to sync with you first).

Reply:
Removed :)